src/php/ext/grpc/php_grpc.c

```diff
@@ -222,6 +232,55 @@ void apply_ini_settings(TSRMLS_D) {
     strcat(poll_str, GRPC_G(poll_strategy));
     putenv(poll_str);
   }
+
+  if (GRPC_G(grpc_verbosity)) {
+    char *verbosity_str = malloc(sizeof("GRPC_VERBOSITY=") +
```

Comment:
Doesn't it need to be freed after use?

Reply:
Oh `putenv` requires the passed buffer live after call. It's up to you to keep it or use `setenv` or `gpr_setenv`.

Reply:
We had issues with `setenv` previously but I forgot exactly why now. I will look it up. Noted about this issue. There are other environment variables that had been set that way. We can address them all together.